tests/onnx/test_onnx_v2.py

```diff
@@ -179,82 +179,79 @@ def test_values_override(self):
+    ("longformer", "allenai/longformer-base-4096"),
+    ("mobilebert", "hf-internal-testing/tiny-random-MobileBertModel"),
     ("mobilenet_v2", "google/mobilenet_v2_0.35_96"),
     ("mobilevit", "apple/mobilevit-small"),
-    ("xlm", "xlm-clm-ende-1024"),
-    ("xlm-roberta", "xlm-roberta-base"),
-    ("layoutlm", "microsoft/layoutlm-base-uncased"),
-    ("layoutlmv3", "microsoft/layoutlmv3-base"),
-    ("groupvit", "nvidia/groupvit-gcc-yfcc"),
-    ("levit", "facebook/levit-128S"),
     ("owlvit", "google/owlvit-base-patch32"),
-    ("vit", "google/vit-base-patch16-224"),
-    ("deit", "facebook/deit-small-patch16-224"),
-    ("beit", "microsoft/beit-base-patch16-224"),
-    ("data2vec-text", "facebook/data2vec-text-base"),
-    ("data2vec-vision", "facebook/data2vec-vision-base"),
-    ("perceiver", "deepmind/language-perceiver", ("masked-lm", "sequence-classification")),
-    ("perceiver", "deepmind/vision-perceiver-conv", ("image-classification",)),
-    ("longformer", "allenai/longformer-base-4096"),
-    ("yolos", "hustvl/yolos-tiny"),
+    ("perceiver", "hf-internal-testing/tiny-random-PerceiverModel", ("masked-lm", "sequence-classification")),
+    ("perceiver", "hf-internal-testing/tiny-random-PerceiverModel", ("image-classification",)),
+    ("resnet", "microsoft/resnet-50"),
+    ("roberta", "hf-internal-testing/tiny-random-RobertaModel"),
+    ("roformer", "hf-internal-testing/tiny-random-RoFormerModel"),
     ("segformer", "nvidia/segformer-b0-finetuned-ade-512-512"),
+    ("squeezebert", "hf-internal-testing/tiny-random-SqueezeBertModel"),
     ("swin", "microsoft/swin-tiny-patch4-window7-224"),
+    ("vit", "google/vit-base-patch16-224"),
+    ("yolos", "hustvl/yolos-tiny"),
     ("whisper", "openai/whisper-tiny.en"),
+    ("xlm", "hf-internal-testing/tiny-random-XLMModel"),
+    ("xlm-roberta", "hf-internal-testing/tiny-random-XLMRobertaXLModel"),
 }
 
 PYTORCH_EXPORT_ENCODER_DECODER_MODELS = {
     ("vision-encoder-decoder", "nlpconnect/vit-gpt2-image-captioning"),
 }
 
 PYTORCH_EXPORT_WITH_PAST_MODELS = {
-    ("bloom", "bigscience/bloom-560m"),
-    ("gpt2", "gpt2"),
-    ("gpt-neo", "EleutherAI/gpt-neo-125M"),
+    ("bloom", "hf-internal-testing/tiny-random-BloomModel"),
+    ("gpt2", "hf-internal-testing/tiny-random-GPT2Model"),
+    ("gpt-neo", "hf-internal-testing/tiny-random-GPTNeoModel"),
 }
 
 PYTORCH_EXPORT_SEQ2SEQ_WITH_PAST_MODELS = {
-    ("bart", "facebook/bart-base"),
-    ("mbart", "sshleifer/tiny-mbart"),
-    ("t5", "t5-small"),
+    ("bart", "hf-internal-testing/tiny-random-BartModel"),
+    ("bigbird-pegasus", "hf-internal-testing/tiny-random-BigBirdPegasusModel"),
+    ("blenderbot-small", "facebook/blenderbot_small-90M"),
+    ("blenderbot", "hf-internal-testing/tiny-random-BlenderbotModel"),
+    ("longt5", "hf-internal-testing/tiny-random-LongT5Model"),
     ("marian", "Helsinki-NLP/opus-mt-en-de"),
+    ("mbart", "sshleifer/tiny-mbart"),
     ("mt5", "google/mt5-base"),
```

Comment:
Same here re the absence of a model tester class to create a tiny-random variant